pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -4571,6 +4571,22 @@ let
         };
       };
 
+      vue.volar = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "volar";
+          publisher = "Vue";
+          version = "2.0.16";
+          hash = "sha256-RTBbF7qahYP4L7SZ/5aCM/e5crZAyyPRcgL48FVL1jk=";
+        };
+        meta = {
+          changelog = "https://marketplace.visualstudio.com/items/Vue.volar/changelog";
```

```suggestion
          changelog = "https://github.com/vuejs/language-tools/blob/master/CHANGELOG.md";
```
The other changelog is just a link to this address.